include/rangers-api/Application/Player/States.h

```diff
@@ -0,0 +1,18 @@
+#pragma once
+
+namespace app::player {
+    class PlayerStateActionBase : public PlayerStateBase {
```

Comment:
Starting braces should be started on a line to keep consistent with the original repository.